src/Tickets/Commerce/Admin_Tables/Orders.php

```diff
@@ -161,7 +164,13 @@ public function prepare_items() {
 
 		$total_items = $orders_repository->found();
 
-		$this->items = $orders_repository->all();
+		$items = $orders_repository->all();
+
+		if ( $orderby && $order ) {
```

Comment:
Would it make sense to change this to `!empty( $orderby )  && !empty( $order) `?

Reply:
It's the same logic. As, `tribe_get_request_var` is already returning `null` so I thought that way was much simple to read.

Reply:
Agreed:
```suggestion
		if ( ! empty( $orderby ) && ! empty( $order ) ) {
```